src/emitter.ts

```diff
@@ -257,7 +268,7 @@ export function emitWebIdl(webidl: Browser.WebIdl, flavor: Flavor) {
                 return event.type;
```

Comment:
why doesn't this one need a call to `getGenericEventType`?

Reply:
They are not affected because they are currently all using FocusEvent. Might be useful in the future though.